Wrong shot number:
[13, 11, 10]

Correct shot number:
[12]

Raw json:

{'hasError': True, 'errParameter': ['alluxio.underfs.s3.threads.max'], 'reason': ["The property 'alluxio.underfs.s3.threads.max' has the value '4000000000' which is out of the valid range of a thread pool size."]}

{'hasError': True, 'errParameter': ['alluxio.underfs.s3.threads.max'], 'reason': ["The value of the property 'alluxio.underfs.s3.threads.max' should be less than 2147483647."]}

{'hasError': True, 'errParameter': ['alluxio.underfs.s3.threads.max'], 'reason': ["The property 'alluxio.underfs.s3.threads.max' has the value '4000000000' which is out of the valid range of threads."]}

Final result:

There are 1 misconfiguration parameters in the input: alluxio.underfs.s3.threads.max
[Ciri] Reason for alluxio.underfs.s3.threads.max: The property 'alluxio.underfs.s3.threads.max' has the value '4000000000' which is out of the valid range of threads.
